tests/MediaStaticTest.php

```diff
@@ -54,16 +54,19 @@ public function testEverythingDisabledExceptDelete(): void
     {
         Route::media(MediaRouteActionEnum::Delete);
 
-        $filePath = 'preview_Private photo';
+        $filePath = 'Private photo';
```

Comment:
let's just test that api return success code for delete enpoint and 404 for all other non registered enpoints